src/components/ExportGithub.tsx

```diff
@@ -23,13 +17,41 @@ type UserReposResponse = {
   ];
   status: number;
 };
-type UserRepos = UserReposResponse["data"];
-const initialRepos: UserRepos = [{ name: "", full_name: "" }];
+export type UserRepos = UserReposResponse["data"];
+export const initialRepos: UserRepos = [{ name: "", full_name: "" }];
+export type GitHubDestination = "Issue" | "File";
```

Comment:
ignore, I see the use case now. To keep in sync with the array below you could instead do:
```
const GITHUB_DESTINATIONS = ["Issue", "File"] as const;
export type GitHubDestination = (typeof GITHUB_DESTINATIONS)[number];
```

makes it so that adding new options is only write-once instead of write-twice